SeaBattle/Sea.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
+        private List<BaseShip> Ships { get; set; }
+        int SeaWidth;
+        int SeaLenght;
+
+        public Sea()
+        {
+            var Ships = new List<BaseShip>();
+        }
+
+
+        public void Create()
+        {
+            //will finish this IF parts tommorow. 
+            //if ( > 10 &&  > 10)
+            //{
+            //}
+
+            for (int i = 0; i < Ships.Count; i++)
```

Comment:
What is piece of logic even needed for? you call a .Create() method on the Sea object, and it fills in ships? Where does they come from? Can you describe the logic of this method?

Reply:
It is not finished FOR, which I wanted to check is that any deck on "point" when adding ship to the sea (but this checks must be in method AddShip() )